accord-core/src/main/java/accord/impl/CommandChange.java

```diff
@@ -503,7 +543,7 @@ public static int getFlags(Command before, Command after)
         }
 
         if (after.saveStatus() != null)
```

Comment:
not changed in this patch, but I think this condition is unnecessary as it is always non-null?